alteryx/featuretools
# v1.5.0

v1.5.0 Feb 14, 2022
===================
**warning**: Featuretools may not support Python 3.7 in next non-bugfix release.

* Enhancements
    * Add ability to use offset alias strings as inputs to rolling primitives (#1809)
    * Update to add support for pandas version 1.4.0 (#1881, #1895)
* Fixes
    * Fix ``featuretools_primitives`` entry point (#1891)
* Changes
    * Allow only snake camel and title case for primitives (#1854)
    * Add autonormalize as an add-on library (#1840)
    * Add DateToHoliday Transform Primitive (#1848)
    * Add DistanceToHoliday Transform Primitive (#1853)
    * Temporarily restrict pandas and koalas max versions (#1863)
    * Add ``__setitem__`` method to overload ``add_dataframe`` method on EntitySet (#1862)
    * Add support for woodwork 0.12.0 (#1872, #1897)
    * Split Datetime and LatLong primitives into separate files (#1861)
    * Null values will not be included in index of normalized dataframe (#1897)
* Documentation Changes
    * Bump ipython version (#1857)
    * Update README.md with Alteryx link (#1886)
* Testing Changes
    * Add check for package conflicts with install workflow (#1843)
    * Change auto approve workflow to use assignee (#1843)
    * Update auto approve workflow to delete branch and change on trigger (#1852)
    * Upgrade tests to use compose version 0.8.0 (#1856)
    * Updated deep feature synthesis and feature serialization tests to use new primitive files (#1861)
